atlasdb-commons/src/main/java/com/palantir/util/AssertUtils.java

```diff
@@ -26,7 +26,18 @@
 
 public class AssertUtils {
 
-    private static final Logger log = LoggerFactory.getLogger(AssertUtils.class);
+    /**
+     * Previously, failed asserts logged to this log.
```

Comment:
how would you shorten it?

I generally bias towards more comments, when the chances of anybody remember this kind of detail 1 year from now is very low, also so folks know when this can go away.